010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,119 @@
+## More than PostgreSQL
 
-```json
-{
-  "name": "John Doe",
-  "email": "john@example.com",
-  "age": 42,
-  "address": "123 Main St, New York",
-  "labels": ["admin", "user"]
-}
-```
+While Xata is built on top of PostgreSQL, it is not a typical PostgreSQL DBaaS, but a _data platform around PostgreSQL_. The following are the charateristics of the data platform.
 
-The above shows a column of type `email` and an array-like column (`labels` of type `multiple`). When this data is stored in PostgreSQL and Elasticsearch, columns are identified by IDs. By using IDs for columns, instead of their user-visible names, we make some migrations simpler and we make it possible, for example, to track a column across branches even if it was renamed.
+### Integrated secondary data stores
 
-As a result, the underlying PostgreSQL table might look something like this:
+While PostgreSQL is an incredibly versitile database, and it's an excelent starting dataabse for almost any application, we are of the opinion that it is not the best tool for every job.
 
-```sql
-  Column  |  Type   | Collation | Nullable | Default | Storage  | Stats target | Description
-----------+---------+-----------+----------+---------+----------+--------------+-------------
- _id      | text    |           | not null |         | extended |              |
- _version | integer |           | not null | 0       | plain    |              |
- a        | text    |           |          |         | extended |              |
- b        | text    |           |          |         | extended |              |
- c        | bigint  |           |          |         | plain    |              |
- d        | boolean |           |          |         | plain    |              |
- e        | text    |           |          |         | extended |              |
- g        | text    |           |          |         | extended |              |
- h        | text[]  |           |          |         | extended |              |
-Indexes:
-"tbl_cd46p95cefoedabj77gg_a_pkey" PRIMARY KEY, btree (_id)
-    "tbl_cd46p95cefoedabj77gg_a_replica_id" UNIQUE, btree (_id, _version) REPLICA IDENTITY
-    "tbl_cd46p95cefoedabj77gg_a_xata_unique_b" UNIQUE CONSTRAINT, btree (b)
-Check constraints:
-    "tbl_cd46p95cefoedabj77gg_a_xata_multiple_length_h" CHECK (octet_length(array_to_string(h, ''::text)) < 65536)
-    "tbl_cd46p95cefoedabj77gg_a_xata_string_length_a" CHECK (length(a) <= 2048)
-    "tbl_cd46p95cefoedabj77gg_a_xata_string_length_b" CHECK (length(b) <= 2048)
-    "tbl_cd46p95cefoedabj77gg_a_xata_string_length_g" CHECK (length(g) <= 2048)
-    "tbl_cd46p95cefoedabj77gg_a_xata_text_length_e" CHECK (octet_length(e) < 204800)
-```
+A good example of this is the File Attachments faeture. While it is possible to store files as BLOBs in PostgreSQL, it would be expensive and inefficient to do so. Instead, we use S3 for file storage and the Cloudflare CDN for file delivery and image transformations.
```

Comment:
```suggestion
A good example of this is the File Attachments feature. While it is possible to store files as BLOBs in PostgreSQL, it would be expensive and inefficient to do so. Instead, we use S3 for file storage and the Cloudflare CDN for file delivery and image transformations.
```